tools_docker/Debian_Uwsgi/send_summary_mails.sh.sample

```diff
@@ -3,4 +3,9 @@ set -a
 . /var/tracim/data/tracim_env_variables_cron
 set +a
 
-<PATH TO tracimcli> periodic send-summary-mails -c <PATH TO development.ini> --since 24
+if [ $1 != "hourly" ] && [ $1 != "daily" ] && [ $1 != "weekly" ]; then
+  echo "Error in send_summary_mails.sh. Argument email_notification_type must be hourly, daily or weekly.";
+  exit 0;
+fi
+
+<PATH TO tracimcli> periodic send-summary-mails -c <PATH TO development.ini> --email_notification_type $1
```

Comment:
I think we can set command directly here with this fix https://github.com/tracim/tracim/pull/6418

We can use tracimcli command directly without path dans config file `-c <PATH TO development.ini>`.

So something like `tracimcli periodic send-summary-mails --email_notification_type $1`

Need also modification in other version:
- Debian_New_Uwsgi
- Debian_Uwsgi_ARM64

Reply:
Mmh indeed, that would be nice
I'll try it out and update if it works

Reply:
it works, I updated the script